get_coverage_Cartesian.py

```diff
@@ -0,0 +1,165 @@
+from html.parser import HTMLParser
+import csv
+import os
+import subprocess
+import shutil
+import fileinput
+from datetime import datetime
+
+jacoco_path = '/target/site/jacoco/index.html'
+current_datetime = datetime.now().strftime("%m.%d.%y_%H:%M:%S")
+log_folder = f"hadoop_puts_logs_{current_datetime}/"
+os.makedirs(log_folder)
+
+class MyHTMLParser(HTMLParser):
+    def __init__(self):
+        super().__init__()
+        self.in_total_td = False
+        self.values = []
+
+    def handle_starttag(self, tag, attrs):
+        if tag == "td" and ("class", "bar") in attrs:
+            self.in_total_td = True
+
+    def handle_data(self, data):
+        if self.in_total_td:
+            value = data.strip().replace(",", "")
+            self.values.append(value)
+
+    def handle_endtag(self, tag):
+        if self.in_total_td and tag == "td":
+            self.in_total_td = False
+
+def get_values_from_html_file(file_path):
+    with open(file_path, 'r', encoding='utf-8') as file:
+        html_content = file.read()
+        parser = MyHTMLParser()
+        parser.feed(html_content)
+        return parser.values
+
+def get_coverage_details_from_html(file_path):
+    values = get_values_from_html_file(file_path)
+#     print(values)
+    missed_instructions, total_instructions = map(int, values[0].split(" of "))
+    missed_branches, total_branches = map(int, values[1].split(" of "))
+    covered_Instructions = total_instructions-missed_instructions
+    covered_Branches = total_branches-missed_branches
+
+#     print("Value of covered_Instructions:", covered_Instructions)
```

Comment:
You can add one customized print to control whether to print or not instead of comment them out.
Example:
```
def print_info(info, enable=false):
       if enable:
             print(info)
```
And use a global variable for the enable